api/airtable.ts

```diff
@@ -0,0 +1,59 @@
+import Airtable from 'airtable'
+import Base from 'airtable/lib/base';
+import Record from 'airtable/lib/record';
+import Table from 'airtable/lib/table';
+
+export class AirtableClient {
+  base: {
+    (tableName: string): Table;
+    _base: Base;
+    getId(): string;
```

Comment:
This interface was just copied by navigating thru references into the Airtable Node module code

Reply:
Looks good! Assuming its the same as accessing other airtable API which seemed to only need BaseId and API key

Reply:
ya it should just be the interface for accessing a table base with its `baseId` and an Airtable API key